tests/unit_tests/test_source_sampling.py

```diff
@@ -0,0 +1,1759 @@
+import os
+import warnings
+import itertools
+
+from operator import itemgetter
+from nose.tools import assert_equal, with_setup, assert_almost_equal, assert_raises
+from random import uniform, seed
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+
+try:
+    from pyne.mesh import Mesh
+    # see if the source sampling module exists but do not import it
+    import imp
+    pyne_info = imp.find_module('pyne')
+    pyne_mod = imp.load_module('pyne', *pyne_info)
+    imp.find_module('source_sampling', pyne_mod.__path__)
+except ImportError:
+    from nose.plugins.skip import SkipTest
+    raise SkipTest
```

Comment:
We don't use `nose` elsewhere -- please use [functionality from pytest](http://doc.pytest.org/en/latest/skipping.html) for skipping tests